app/extensions/config/resources.py

```diff
@@ -47,25 +45,3 @@ def patch(self, args):
         """
         response = parameters.PatchHoustonConfigParameters.perform_patch(args, obj=None)
         return response
-
```

Comment:
moved to site settings so that all settings that are actually used by the frontend are in one place